CMakeLists.txt

```diff
@@ -158,7 +162,9 @@ ENDIF ()
 INSTALL (TARGETS msgpack msgpack-static DESTINATION lib)
 INSTALL (DIRECTORY src/msgpack DESTINATION include)
 INSTALL (FILES src/msgpack.h src/msgpack.hpp DESTINATION include)
-INSTALL (FILES msgpack.pc DESTINATION lib/pkgconfig)
+IF(NOT MSVC)
+	INSTALL (FILES msgpack.pc DESTINATION lib/pkgconfig)
+ENDIF()
```

Comment:
It looks fine. But I've never do 'make install' on Windows environment. The original code copied one file that is not needed on Windows. So I think that it is not a critical issue but OK to merge.